src/SourceBuild/tarball/content/src/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+
+  <!--
+    Prevent automatic Directory.Build imports from finding source-build infra. Note: used for
```

Comment:
My preference is for the message to favor readers within the tarball context.  The `Note` is confusing IMO in this context because they don't know how it was created.  I would drop it and maybe add something like `repo src` to the first sentence.

> Prevent repo src automatic Directory.Build imports from finding source-build infra.